docs/02-conda_envs.rst

```diff
@@ -0,0 +1,115 @@
+An Astropy User's Guide to Managing Conda Environments and Jupyter Notebook
+===========================================================================
+
+
+    Help! My code library works in Python 2 and I don't have time to
+    make it work in Python 3
+
+Do not fear! This tutorial will teach you how to use conda environments
+to create a *separate* installation of Python 2.7. With conda
+environments, you can switch between Python 2 and 3 without having to
+worry about version conflicts.
+
```

Comment:
This is *probably* the place to put the "why the py3 switch is happening", right @eblur?  If so, here's some wording:

```
Note that when you have time you probably should try to port your code to Python 3.  Most of the Python scientific software ecosystem has moved from Python 2 to Python 3, because the Python core developers have not added new features to Python 2 since 2010, and are ceasing to support Python 2 entirely at the end of 2019.  This means things like security patches or compiler fixes will stop being issued for Python 2 at that time.  So Python 2 will become harder and harder to safely install after that.  Moreover, Python 3 is both more internally consistent and more feature-ful than Python 2.  For some discussion on how to convert scientific code to Python 3, see: https://python-3-for-scientists.readthedocs.io/en/latest/
```
Does that seem good?